src/plugins/internal/InternalInstallPlugin.ml

```diff
@@ -146,9 +77,12 @@ let install pkg argv =
       fun fn -> fn
   in
 
-  let install_file ?tgt_fn src_file envdir =
+  let install_file ?(with_destdir=true) ?tgt_fn src_file envdir =
```

Comment:
I agree that right now, there is no obvious difference. I just want to have a single piece of code to compute location in findlib (i.e. rely on ocamlfind calls).

Long term, using "ocamlfind install" to compute root/subdirectories will just make the code future proof. If a change in location happen, we will still be in sync.

Concerning the fact that I still prefer using "ocamlfind install" for the root directory:
- Installing everything in the same directory is actually the norm (because you have no easy way to do otherwise...), this is the general case
- Installing in subdirectories is the special case.

I prefer to use standard install (ocamlfind install) for the general case and the new install (files install) for the special case. There will be no change on the general case code path and a change on the new one (activated by the feature). This is a safe approach.

Long term, if you still feel strong about it and the feature is no more Alpha/Beta and works great in every case, we can change this (i.e. feel free to add a "TODO: consider removing "ocamlfind install" when feature X will be official).